### PR TITLE
Added Text for the contributing file

### PR DESCRIPTION
**Pull request by angelorodem**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+the final release.
+
+# Code reviews
+All submissions, including submissions by project members, require review. We
+use Github pull requests for this purpose.
+
+Some tips for good pull requests:
+* Use our code
+  [style guide](https://www.python.org/dev/peps/pep-0008/).
+  When in doubt, try to stay true to the existing code of the project.
```
Comment (jvlsg) [inline CONTRIBUTING.md:47]: The existing code sucks tho, i recommend
"Try your best :)"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+* Write a descriptive commit message. What problem are you solving and what
```
Comment (jvlsg) [inline CONTRIBUTING.md:48]: List changes in the commit mesage using something similar to the following:
UPD for general updates
FIX for clear bug fixes
ADD for new features, files, classes
DEL for deleted features, files, classes

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+  are the consequences? Where and what did you test? Some good tips:
+  [here](http://robots.thoughtbot.com/5-useful-tips-for-a-better-commit-message)
+  and [here](https://www.kernel.org/doc/Documentation/SubmittingPatches).
+* If your PR consists of multiple commits which are successive improvements /
+  fixes to your first commit, consider squashing them into a single commit
+  (`git rebase -i`) such that your PR is a single commit on top of the current
+  HEAD. This make reviewing the code so much easier, and our history more
+  readable.
+* Make sure that all tests are passing before approval.
+* Apply the appropriate language label (e.g., C++, Java, Python, etc.) to the
```
Comment (jvlsg) [inline CONTRIBUTING.md:58]: Remove language label, we never used it

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+hear from you for 2 weeks or longer, we may close the pull request. You can
+still send the pull request again once you have time to work on it.
+
+Once a pull request is merged, we will take care of the rest and get it into
+the final release.
+
+# Code reviews
+All submissions, including submissions by project members, require review. We
+use Github pull requests for this purpose.
+
```
Comment (jvlsg) [inline CONTRIBUTING.md:43]: move the line "## Pull Request Guidelines" here

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+* Provide a good PR description as a record of what change is being made and
+  why it was made. Link to a GitHub issue if it exists.
+* Don't fix code style and formatting unless you are already changing that
+  line to address an issue. PRs with irrelevant changes won't be merged. If
+  you do want to fix formatting or style, do that in a separate PR.
+* Unless your PR is trivial, you should expect there will be reviewer comments
+  that you'll need to address before merging. We expect you to be reasonably
+  responsive to those comments, otherwise the PR will be closed after 2-3 weeks
+  of inactivity.
+* Maintain clean commit history and use meaningful commit messages. PRs with
```
Comment (jvlsg) [inline CONTRIBUTING.md:81]: Remove this item - this is already covered by an item above "If your PR consists of multiple commits which are..."

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+  everyone will be happy.
+* For speculative changes, consider opening an issue and discussing it first.
+  If you are suggesting a behavioral or API change, make sure you get explicit
+  support from a DSHC team member before sending us the pull request.
+* Provide a good PR description as a record of what change is being made and
+  why it was made. Link to a GitHub issue if it exists.
+* Don't fix code style and formatting unless you are already changing that
+  line to address an issue. PRs with irrelevant changes won't be merged. If
+  you do want to fix formatting or style, do that in a separate PR.
+* Unless your PR is trivial, you should expect there will be reviewer comments
```
Comment (jvlsg) [inline CONTRIBUTING.md:77]: Remove this item - under "Contributing process" we describe the wait times for PRs

File: CONTRIBUTING.md (hunk 2 above)
Comment (angelorodem) [reply CONTRIBUTING.md:46]: added at the end

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+### Contributor License Agreements
+Contributions to this project will be automatically be assumed as GNU LESSER GENERAL PUBLIC LICENSE (This project license).
+
+### Coding Style
+This project *aims* to follows [PEP-8 Coding Style Guides](https://www.python.org/dev/peps/pep-0008/).
+Before sending out your pull request, please familiarize yourself with the
+corresponding style guides and make sure the proposed code change is style
+conforming.
+
+## Contributing Process
```
Comment (jvlsg) [inline CONTRIBUTING.md:20]: Make this a Heading 1
`# Contributing Process`

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+
+## Before You Start
+We accept patches in the form of github pull requests. If you are new to
+github, please read [How to create github pull requests](https://help.github.com/articles/about-pull-requests/)
+first.
+
+### Contributor License Agreements
+Contributions to this project will be automatically be assumed as GNU LESSER GENERAL PUBLIC LICENSE (This project license).
+
+### Coding Style
```
Comment (jvlsg) [inline CONTRIBUTING.md:14]: Place the Coding Style under "Contributing process" as a H2
```
# Contributing Process
....
## Coding Style
```

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+* Keep your PR up to date with upstream/master (if there are merge conflicts,
+  we can't really merge your change).
+* All tests need to be passing before your change can be merged. We recommend
+  you run tests locally before creating your PR to catch breakages early on.
+  Ultimately, the green signal will be provided by our testing infrastructure.
+  The reviewer will help you if there are test failures that seem not related
+  to the change you are making.
+
+
+## Commit Guideline
```
Comment (jvlsg) [inline CONTRIBUTING.md:77]: Place the Commit Guideline under "Contributing process" as a H2
```
# Contributing Process
## Coding Style
## Commit Guideline
```

**Pull request merged**